packages/mds-compliance/mds-compliance-engine.ts

```diff
@@ -276,4 +274,10 @@ function filterPolicies(policies: Policy[]): Policy[] {
   })
 }
 
-export { processPolicy, filterPolicies }
+function filterEvents(events: VehicleEvent[], end_time = now()): VehicleEvent[] {
+  return events.filter((event: VehicleEvent) => {
+    return event.timestamp > end_time - TWO_DAYS_IN_MS
```

Comment:
recommend a comment for the intent, e.g. `keep events that are less than two days old`